File: C_Programming/Day_5/program38.c
/* 
    Description : Write a program that accepts frequency & prints
                  the below pattern : 
    Input : 4  4
    Output :1 * 3 * 
            1 * 3 *
            1 * 3 * 
            1 * 3 *
*/

#include<stdio.h>

void Display(int iRow , int iCol)
{
    int i = 0 ;
    int j = 0;

    for(i = 1 ; i <= iRow ; i++)
    {
        for(j = 1 ; j <= iCol ; j++)
        {
            if(j % 2 != 0)
            {
                printf("%d\t",j);
            }
            else
            {
                printf("*\t");
            }

        }
        printf("\n");
    }

}


int main()
{
    int iRow = 0;
    int iCol = 0;
    char ch = '\0';

    printf("Enter the number of rows : ");
    scanf("%d",&iRow);

    printf("Enter the number of columns : ");
    scanf("%d",&iCol);

    Display(iRow , iCol);

    return 0;
}

File: Python_Programming/Day_5/program38.py
'''
    Description : Write a program that accprts frequency & prints
                  the below pattern : 
    Input : 4  4
    Output :1 * 3 * 
            1 * 3 *
            1 * 3 * 
            1 * 3 *
'''
def Display(Row,Col):

    for i in range(1 , Row+1 , 1):
        for j in range(1 , Col+1 , 1):
            if(j % 2 != 0):
                print(j,"\t" , end="")
            else:
                print("*\t" , end="")
        print('\n')



def main():
    Row = 0
    Col = 0

    print("Enter the number of rows : ",end = "")
    Row = int(input())

    print("Enter the number of columns : ",end = "")
    Col = int(input())

    Display(Row , Col)

if __name__ == "__main__":
    main()